client/src/enums/nr-state.ts

```diff
@@ -0,0 +1,12 @@
+/** Enum for NR State Code. */
+export enum NrState {
+  APPROVED = 'APPROVED',
+  CANCELLED = 'CANCELLED',
+  CONDITIONAL = 'CONDITIONAL',
+  DRAFT = 'DRAFT',
+  EXPIRED = 'EXPIRED',
+  IN_PROGRESS = 'INPROGESS',
+  ON_HOLD = 'ONHOLD', // TODO: verify this
+  REFUNDED = 'REFUNDED', // TODO: verify this
```

Comment:
I need to test these. Does anyone have a NR that's On Hold, and one that's Refunded? (I have asked John for these as well.)

Reply:
I think* if you ask Sienna or Janis they can set any state you like, once you have an NR in draft.

Reply:
Thanks for the tip (in case John falls through ;) ).